### PR TITLE
Release for v.0.15.0

### PR DESCRIPTION
**Pull request by canyon289**
## Description
Prep v0.15.0 release

----
:books: Documentation preview :books:: https://arviz--2206.org.readthedocs.build/en/2206/

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Change Log
 
-## v0.x.x Unreleased
+## v0.15.0 Unreleased
```
Comment (canyon289) [inline CHANGELOG.md:3]:
```suggestion
## v0.15.0 Released
```

**Review by aloctavodia — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Change Log
 
-## v0.x.x Unreleased
+## v0.15.0 Released
```
Comment (OriolAbril) [inline CHANGELOG.md:3]: This should be the date it was released on

**Review by OriolAbril — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (canyon289) [reply CHANGELOG.md:3]: Will set that when I release tommorrow!

Comment (canyon289) [reply CHANGELOG.md:3]: On Friday*

Comment (OriolAbril) [reply CHANGELOG.md:3]: Perfect, thanks

Comment (OriolAbril) [reply CHANGELOG.md:3]: In case it helps, we also have https://python.arviz.org/en/latest/contributing/how_to_release.html

**Comment by canyon289** [comment]
Everything passed in Azure. Not sure why the status isnt updating here

image

**Comment by canyon289** [comment]
All changes addressed. Merging

**Pull request merged**